### PR TITLE
Fy 04 explore page

### LINKED ISSUE #7: Feat: "Explore Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
This PR Contains
Explore page
Single Video Page
Added Category Filters
Filtering Videos based on Category
Added Videos in the DB

### CODE REVIEW

File: src/components/Loader/Loader.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    margin: 0 auto;
+  `;
+  return (
+    <>
+      <PulseLoader
+        color={"#FFF"}
+        loading={loading}
+        css={override}
+        size={15}
+        margin={3}
```
Comment (nishasen) [inline src/components/Loader/Loader.jsx:15]: Good use of loader

**Review by nishasen — APPROVED** [review]
Well structured code, good use of some packages.

File: src/components/Loader/Loader.jsx (hunk 1 above)
Comment (sadathshariff) [reply src/components/Loader/Loader.jsx:15]: Thanks Nisha

**Review by aishwarya-bm — APPROVED** [review]
Looks all good, i liked how you broke the components and great usage of utils

**Pull request merged**